common/src/main/java/org/broadleafcommerce/common/i18n/service/TranslationServiceImpl.java

```diff
@@ -492,7 +510,7 @@ protected boolean propertyInDefaultLocaleExceptionList(Object entity, String pro
      * @param locale
      * @return
      */
-    protected boolean localeMatchesDefaultLocale(Locale locale) {
+    protected boolean languageMatchesDefaultLocaleLanguage(Locale locale) {
```

Comment:
This is an API change that could break clients.  We probably don't want to have this as part of the PR